docs/3.0.0-beta.x/concepts/concepts.md

```diff
@@ -216,14 +222,34 @@ Please refer to the [internationalization's guide](../guides/i18n.md).
 
 ## Plugin
 
-A plugin is like a fully independent sub-application. It has its own business logic with dedicated models, controllers, services, middlewares or hooks. It can also contain an UI integrated into the admin panel to use it easily. It allows to develop or plugin features in a project in a short time span.
+A plugin is like a small independent sub-application. It has its own business logic with dedicated models, controllers, services, middlewares or hooks. It can also have it's own UI integrated in the admin panel.
 
 ::: note
 Please refer to the [plugins documentation](../plugin-development/quick-start.md) for more informations.
 :::
 
 ---
 
+## Extensions
+
+In strapi you can install plugins in your `node_modules`. This allows for easy updates and respect best practices. To customize those installed plugins you can work in the `/extensions` directory. It contains all the plugins' customizable files.
+
+Certain plugins will create files in these folders so you can then modify them. You can also create certain files manually to add some custom configuration for example.
```

Comment:
Are we saying, 

"Certain plugins will files in this `/extensions` directory providing an ability to modify them. You may also create certain configuration type files and add these to this directory."